src/test/codegen/target-feature-multiple.rs

```diff
@@ -0,0 +1,9 @@
+// only-x86_64
+// compile-flags: -C target-feature=+sse2,-avx,+avx2 -C target-feature=+avx,-avx2
+
+#![crate_type = "lib"]
+
+#[no_mangle]
+pub fn foo() {
+    // CHECK: attributes #0 = { {{.*}}"target-features"="+sse2,-avx,+avx2,+avx,-avx2"{{.*}} }
```

Comment:
See https://github.com/rust-lang/rust/issues/44815#issuecomment-626384743 regarding canonicalization of the feature list.
I'll try to fix this some time later together with other issues.